EventListener/ExceptionListener.php

```diff
@@ -53,14 +58,19 @@ public function onKernelException($event)
         $request = $event->getRequest();
 
         if (!$request->attributes->get(FOSRestBundle::ZONE_ATTRIBUTE, true)) {
-            return;
+            if (null === $this->innerExceptionListener) {
+                return;
+            }
+
+            return $this->innerExceptionListener->onKernelException($event);
         }
 
         $exception = $event->getThrowable();
 
         $controllerArgsListener = function ($event) use (&$controllerArgsListener, $exception) {
```

Comment:
this code can be removed safely if the exception argument of the ExceptionController is typed as `\Throwable`.

Reply:
hmm removing this I get failures in the test suite .. didn't have time to investigate if these are just problems in the test suite. also not sure if we can do some tweaks to the ExceptionController to make it work without this code .. can you make your suggesting more concrete?

Reply:
Yes, I send you a concrete example in this comment https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2108#discussion_r387722224 :)

Reply:
ah sorry .. can you send this as a PR into this PR?

Reply:
see https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2128